### PR TITLE
Add Release Notes for iOS Release 1.58

### LINKED ISSUE #8487: Release Notes for iOS Release 1.58
### Web3

- Added Coinbase Pay on-ramp for Brave Wallet buy flows. [(#7880](https://github.com/brave/brave-ios/issues/7880))
- Added Stripe on-ramp for Brave Wallet buy flows. ([#7794](https://github.com/brave/brave-ios/issues/7794))
- Added Filecoin support in Brave Wallet. ([#7809](https://github.com/brave/brave-ios/issues/7809))
- Added the ability for users to choose which Solana account to choose when connecting to Solana DApps. ([#7637](https://github.com/brave/brave-ios/issues/7637))
- Updated default buy currency and removed field placeholder for Stripe on ramp flow. ([#7709](https://github.com/brave/brave-ios/issues/7709))
- Refactored Selected Account in Brave Wallet. ([#7723](https://github.com/brave/brave-ios/issues/7723))
- Fixed backup flow to show as completed when initiated from “Accounts” tab. ([#8120](https://github.com/brave/brave-ios/issues/8120))
- Fixed Solana DApps to disconnect when permission is revoked. ([#7688](https://github.com/brave/brave-ios/issues/7688))

### General 

- Implemented “Copy clean URL” feature on iOS. ([#6179](https://github.com/brave/brave-ios/issues/6179))
- Added support for Apple Screentime. ([#3939](https://github.com/brave/brave-ios/issues/3939))
- Added support for more regions for Brave News opt-in card. ([#7932](https://github.com/brave/brave-ios/issues/7932))
- Added support for Playlist on https://www.bitchute.com. ([#7916](https://github.com/brave/brave-ios/issues/7916))
- Added “Suppress” button to “Allow link to switch apps?” pop-up as reported on HackerOne by amit5533.  [#7835](https://github.com/brave/brave-ios/issues/7835)
- Added support for Playlist on https://www.x.com. ([#7896](https://github.com/brave/brave-ios/issues/7896))
- Added logic for device limitations when joining existing Sync chain. ([#7887](https://github.com/brave/brave-ios/issues/7887))
- Added throttling when compiling filter lists in parallel to reduce crash rates. ([#8191](https://github.com/brave/brave-ios/issues/8191))
- Improved performance of compiling engines for cosmetic filtering. ([#7955](https://github.com/brave/brave-ios/issues/7955))
- Removed bookmark count for bookmark folders. ([#8041](https://github.com/brave/brave-ios/issues/8041))
- Removed Zip Foundation dependency. ([#7901](https://github.com/brave/brave-ios/issues/7901))
- Removed a script on YouTube which caused ads to be shown. ([#8239](https://github.com/brave/brave-ios/issues/8239))
- Updated button text in the Privacy Preserving Product Analytics (P3A) onboarding screen. ([#8056](https://github.com/brave/brave-ios/issues/8056))
- Updated logic and strings for “Set Brave as your default browser” notification. ([#8045](https://github.com/brave/brave-ios/issues/8045))
- Updated VPN icon in all locations within the app. ([#7995](https://github.com/brave/brave-ios/issues/7995))
- Updated User Agent to version iOS 16.6. ([#7987](https://github.com/brave/brave-ios/issues/7987))
- Updated “Storage almost full” message to be shown once per day when adding to Playlist. ([#7869](https://github.com/brave/brave-ios/issues/7869))
- Updated Jitsi SDK to version 8.3.1-lite. ([#7940](https://github.com/brave/brave-ios/issues/7940))
- Fixed a crash when upgrading app with all filter lists enabled. ([#8277](https://github.com/brave/brave-ios/issues/8277))
- Fixed an intermittent crash caused by taking screenshots of tabs. ([#8268](https://github.com/brave/brave-ios/issues/8268))
- Fixed app from crash when changing VPN regions after switching to IKEv2 protocol. ([#8235](https://github.com/brave/brave-ios/issues/8235))
- Fixed crash on launch when using a language unsupported by Chromium. ([#8213](https://github.com/brave/brave-ios/issues/8213))
- Fixed tab view not switching on iPad when using the plus sign button in Private Mode. ([#8241](https://github.com/brave/brave-ios/issues/8241))
- Fixed content leaking in Private windows when switching windows (iPad). ([#8228](https://github.com/brave/brave-ios/issues/8228))
- Fixed unwanted removal of content blockers during browser launch. ([#8219](https://github.com/brave/brave-ios/issues/8219))
- Fixed issue with not applying aggressive only regional filter lists. ([#8218](https://github.com/brave/brave-ios/issues/8218))
- Fixed hamburger menu not opening when using Split View (iPad). ([#8183](https://github.com/brave/brave-ios/issues/8183))
- Fixed “Set Brave as your default browser” notification still being shown after opening external link. ([#8163](https://github.com/brave/brave-ios/issues/8163))
- Fixed problem with the “Bookmarks” switch in Sync Settings being disabled when enabling other switches. ([#8148](https://github.com/brave/brave-ios/issues/8148))
- Fixed “Switch to App” notices being shown on https://www.linkedin.com and https://www.instagram.com. ([#8066](https://github.com/brave/brave-ios/issues/8066))
- Fixed tab order being lost for normal tabs after toggling Private Mode. ([#7996](https://github.com/brave/brave-ios/issues/7996))
- Fixed “Reopen Last Closed Tab”  from being shown when no tabs have been closed. ([#7897](https://github.com/brave/brave-ios/issues/7897))
- Fixed spinner on VPN purchase screen when using "restore" button. ([#7875](https://github.com/brave/brave-ios/issues/7875))
- Fixed YouTube videos from continuing to play after closing tab. ([#7871](https://github.com/brave/brave-ios/issues/7871))
- Fixed tab being selected in Tabs bar after loading website. ([#7851](https://github.com/brave/brave-ios/issues/7851))
- Fixed Reader Mode bar background color to match the selected app UI theme.  ([#7717](https://github.com/brave/brave-ios/issues/7717))
- Fixed intermittent crash when reordering tabs. ([#7645](https://github.com/brave/brave-ios/issues/7645))
- Fixed Private tab to show as blank page when using the app switcher. ([#6577](https://github.com/brave/brave-ios/issues/6577))

### PR DESCRIPTION
**Pull request by Uni-verse**
## Summary of Changes

This pull request fixes [#8487](https://github.com/brave/brave-ios/issues/8487)

## Submitter Checklist:

- [ ] *Unit Tests* are updated to cover new or changed functionality
- [ ] User-facing strings use `NSLocalizableString()`
- [ ] New or updated UI has been tested across:
  - [ ] Light & dark mode
  - [ ] Different size classes (iPhone, landscape, iPad)
  - [ ] Different dynamic type sizes

## Test Plan:

N/A

## Reviewer Checklist:

- [ ] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `bug` / `enhancement`
- [ ] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [ ] Adequate unit test coverage exists to prevent regressions.
- [ ] Adequate test plan exists for QA to validate (if applicable).
- [ ] Issue and pull request is assigned to a milestone (should happen at merge time).

### CODE REVIEW

**Review by iccub — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,59 @@
 # Changelog
 
+## [1.58](https://github.com/brave/brave-ios/releases/tag/v1.58)
+
+### Web3
+
+- Added Coinbase Pay on-ramp for Brave Wallet buy flows. [(#7880](https://github.com/brave/brave-ios/issues/7880))
```
Comment (srirambv) [inline CHANGELOG.md]: Maybe reword as  `Added Coinbase Pay as an On-ramp provider.`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,59 @@
...
+- Added Stripe on-ramp for Brave Wallet buy flows. ([#7794](https://github.com/brave/brave-ios/issues/7794))
```
Comment (srirambv) [inline CHANGELOG.md]: Maybe reword as  `Added Stripe as an On-ramp provider.`

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,59 @@
...
+- Added Filecoin support in Brave Wallet. ([#7809](https://github.com/brave/brave-ios/issues/7809))
+- Added the ability for users to choose which Solana account to choose when connecting to Solana DApps. ([#7637](https://github.com/brave/brave-ios/issues/7637))
```
Comment (srirambv) [inline CHANGELOG.md]: Maybe reword to `Added ability to choose Solana account when connecting to Solana DApp.`

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,59 @@
...
+- Updated default buy currency and removed field placeholder for Stripe on ramp flow. ([#7709](https://github.com/brave/brave-ios/issues/7709))
```
Comment (srirambv) [inline CHANGELOG.md]: Maybe reword to `Removed placeholder value in "Buy" screen and updated default currency to USD`

Placeholder is removed for all on-ramp not just for Stripe so no need to mention the provider here.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,59 @@
...
+- Refactored Selected Account in Brave Wallet. ([#7723](https://github.com/brave/brave-ios/issues/7723))
```
Comment (srirambv) [inline CHANGELOG.md]: Marked it as exclude so can remove it

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,59 @@
...
+- Fixed backup flow to show as completed when initiated from “Accounts” tab. ([#8120](https://github.com/brave/brave-ios/issues/8120))
```
Comment (srirambv) [inline CHANGELOG.md]: Change to `Fixed wallet backup flow  when initiated from "Accounts" tab`

**Review by srirambv — CHANGES_REQUESTED** [review]

**Comment by Uni-verse** [comment]
Thanks for the review @srirambv 

https://github.com/brave/brave-ios/commit/76f2640abe4f90df655f77a481afa756a0b8b5c9

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,58 @@
...
+- Added Stripe as an On-ramp provider. ([#7794](https://github.com/brave/brave-ios/issues/7794))
+- Added Filecoin support in Brave Wallet. ([#7809](https://github.com/brave/brave-ios/issues/7809))
+- Added ability to choose Solana account when connecting to Solana DApp. ([#7637](https://github.com/brave/brave-ios/issues/7637))
+- Removed placeholder value in "Buy" screen and updated default currency to USD. ([#7709](https://github.com/brave/brave-ios/issues/7709))
+- Fixed wallet backup flow when initiated from "Accounts" tab. ([#8120](https://github.com/brave/brave-ios/issues/8120))
+- Fixed Solana DApps to disconnect when permission is revoked. ([#7688](https://github.com/brave/brave-ios/issues/7688))
+
+### General 
+
+- Implemented “Copy clean URL” feature on iOS. ([#6179](https://github.com/brave/brave-ios/issues/6179))
```
Comment (kjozwiak) [inline CHANGELOG.md]: Probably don't need to mention iOS as the CHANGELOG/Release notes are specifically for iOS in this case.

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,58 @@
...
+- Added support for Apple Screentime. ([#3939](https://github.com/brave/brave-ios/issues/3939))
+- Added support for more regions for Brave News opt-in card. ([#7932](https://github.com/brave/brave-ios/issues/7932))
+- Added support for Playlist on https://www.bitchute.com. ([#7916](https://github.com/brave/brave-ios/issues/7916))
+- Added “Suppress” button to “Allow link to switch apps?” pop-up as reported on HackerOne by amit5533.  [#7835](https://github.com/brave/brave-ios/issues/7835)
```
Comment (kjozwiak) [inline CHANGELOG.md]: nit: extra space after the period (before issue link)

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,58 @@
...
+- Added support for Playlist on https://www.x.com. ([#7896](https://github.com/brave/brave-ios/issues/7896))
```
Comment (kjozwiak) [inline CHANGELOG.md:21]: Maybe add this below the https://www.bitchute.com so both `Playlist` support issue are together?

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,5 +1,58 @@
...
+- Added logic for device limitations when joining existing Sync chain. ([#7887](https://github.com/brave/brave-ios/issues/7887))
```
Comment (kjozwiak) [inline CHANGELOG.md]: nit: Sync probably doesn't need to be capitalized in this instance unless using `Brave Sync`.

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,5 +1,58 @@
...
+- Added throttling when compiling filter lists in parallel to reduce crash rates. ([#8191](https://github.com/brave/brave-ios/issues/8191))
+- Improved performance of compiling engines for cosmetic filtering. ([#7955](https://github.com/brave/brave-ios/issues/7955))
+- Removed bookmark count for bookmark folders. ([#8041](https://github.com/brave/brave-ios/issues/8041))
+- Removed Zip Foundation dependency. ([#7901](https://github.com/brave/brave-ios/issues/7901))
```
Comment (kjozwiak) [inline CHANGELOG.md]: Can probably remove this one TBH. I don't mind keeping it but don't think it will add much value to regular users.

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,5 +1,58 @@
...
+- Removed a script on YouTube which caused ads to be shown. ([#8239](https://github.com/brave/brave-ios/issues/8239))
```
Comment (kjozwiak) [inline CHANGELOG.md]: Rather than mentioning scripts being removed, can do something more general:

```
- Fixed ads being shown on YouTube in certain cases.
```

File: CHANGELOG.md [hunk 13]
```diff
@@ -1,5 +1,58 @@
...
+- Updated button text in the Privacy Preserving Product Analytics (P3A) onboarding screen. ([#8056](https://github.com/brave/brave-ios/issues/8056))
+- Updated logic and strings for “Set Brave as your default browser” notification. ([#8045](https://github.com/brave/brave-ios/issues/8045))
+- Updated VPN icon in all locations within the app. ([#7995](https://github.com/brave/brave-ios/issues/7995))
+- Updated User Agent to version iOS 16.6. ([#7987](https://github.com/brave/brave-ios/issues/7987))
+- Updated “Storage almost full” message to be shown once per day when adding to Playlist. ([#7869](https://github.com/brave/brave-ios/issues/7869))
+- Updated Jitsi SDK to version 8.3.1-lite. ([#7940](https://github.com/brave/brave-ios/issues/7940))
+- Fixed a crash when upgrading app with all filter lists enabled. ([#8277](https://github.com/brave/brave-ios/issues/8277))
+- Fixed an intermittent crash caused by taking screenshots of tabs. ([#8268](https://github.com/brave/brave-ios/issues/8268))
+- Fixed app from crash when changing VPN regions after switching to IKEv2 protocol. ([#8235](https://github.com/brave/brave-ios/issues/8235))
```
Comment (kjozwiak) [inline CHANGELOG.md]: Maybe something like this?

```
- Fixed crash when changing VPN regions after switching to IKEv2 protocol. ([#8235](https://github.com/brave/brave-ios/issues/8235))
```

File: CHANGELOG.md [hunk 14]
```diff
@@ -1,5 +1,58 @@
...
+- Fixed crash on launch when using a language unsupported by Chromium. ([#8213](https://github.com/brave/brave-ios/issues/8213))
+- Fixed tab view not switching on iPad when using the plus sign button in Private Mode. ([#8241](https://github.com/brave/brave-ios/issues/8241))
+- Fixed content leaking in Private windows when switching windows (iPad). ([#8228](https://github.com/brave/brave-ios/issues/8228))
+- Fixed unwanted removal of content blockers during browser launch. ([#8219](https://github.com/brave/brave-ios/issues/8219))
+- Fixed issue with not applying aggressive only regional filter lists. ([#8218](https://github.com/brave/brave-ios/issues/8218))
+- Fixed hamburger menu not opening when using Split View (iPad). ([#8183](https://github.com/brave/brave-ios/issues/8183))
+- Fixed “Set Brave as your default browser” notification still being shown after opening external link. ([#8163](https://github.com/brave/brave-ios/issues/8163))
+- Fixed problem with the “Bookmarks” switch in Sync Settings being disabled when enabling other switches. ([#8148](https://github.com/brave/brave-ios/issues/8148))
```
Comment (kjozwiak) [inline CHANGELOG.md]: nit: Sync probably doesn't need to be capitalized in this instance unless using `Brave Sync`. Lets add `"Settings"` in quotes as well.

File: CHANGELOG.md [hunk 15]
```diff
@@ -1,5 +1,58 @@
...
+- Added ability to choose Solana account when connecting to Solana DApp. ([#7637](https://github.com/brave/brave-ios/issues/7637))
+- Removed placeholder value in "Buy" screen and updated default currency to USD. ([#7709](https://github.com/brave/brave-ios/issues/7709))
+- Fixed wallet backup flow when initiated from "Accounts" tab. ([#8120](https://github.com/brave/brave-ios/issues/8120))
+- Fixed Solana DApps to disconnect when permission is revoked. ([#7688](https://github.com/brave/brave-ios/issues/7688))
+
+### General 
+
+- Implemented “Copy clean URL” feature on iOS. ([#6179](https://github.com/brave/brave-ios/issues/6179))
+- Added support for Apple Screentime. ([#3939](https://github.com/brave/brave-ios/issues/3939))
+- Added support for more regions for Brave News opt-in card. ([#7932](https://github.com/brave/brave-ios/issues/7932))
```
Comment (kjozwiak) [inline CHANGELOG.md]: I think we can just say that we added Brave News support for those specific regions. For example:

```
- Added Brave News support for "Germany", "France", "India" & "Austria".
```

**Review by kjozwiak — CHANGES_REQUESTED** [review]

**Review by srirambv — APPROVED** [review]
++ for Web3

**Comment by Uni-verse** [comment]
@kjozwiak  Appreciate the review.

**Review by kjozwiak — APPROVED** [review]
LGTM 👍 Awesome work.
